lib/passpartu.rb

```diff
@@ -28,22 +30,50 @@ def self.configure
 
   class Config
     attr_accessor :policy, :raise_policy_missed_error
-    attr_reader :policy_file
+    attr_reader :policy_file, :use_custom_config_file
+    PASSPARTU_HOME = File.realpath(File.join(File.dirname(__FILE__), '..'))
+    PASSPARTU_CONFIG_TEMPLATE = File.join(PASSPARTU_HOME, 'config', 'passpartu_template.yml')
+    PASSPARTU_INITIALIZER_TEMPLATE = File.join(PASSPARTU_HOME, 'config', 'initializers', 'passpartu.rb')
+    APP_CONFIG_FILE_PATH = './config/passpartu.yml'
+    APP_INITIALIZER_PATH = './config/initializers/passpartu.rb'
 
```

Comment:
Move all new constants and methods that are related to initial file generation to new module `initializer.rb`
Rename constants
`PASSPARTU_CONFIG_TEMPLATE ` -> `CONFIG_FILE_TEMPLATE`
`PASSPARTU_INITIALIZER_TEMPLATE ` -> `INITITALIZER_FILE_TEMPALTE`
`PASSPARTU_HOME ` -> `HOME_PATH`
`APP_CONFIG_FILE_PATH ` -> `DEFAULT_CONFIG_FILE_PATH `
`APP_INITIALIZER_PATH ` -> `DEFAULT_INITIALIZER_PATH `

Check and create file using one method from new module, something like: `check_or_create_defaults`